File: Electronics_Shop_17.Model/Requests/AddAdress.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Model.Requests
{
    public class AddAdress
    {
        public string Street { get; set; }
        public string City { get; set; }
        public string Country { get; set; }
        public string PostalCode { get; set; }

        public int PersonId { get; set; }
        public int CustomerId { get; set; }
        public bool isDeleted { get; set; } = false;
    }
}

File: angular17_web/src/app/models/adress/add.adress.ts
export class AddAdress {
  street!: string;
  city!: string;
  country!: string;
  postalCode!: string;
  personId!: number;
  customerId!: number;
  isDeleted?: boolean;

  constructor(data?: any) {
    if (!data) return;

    this.street = data.street;
    this.city = data.city;
    this.country = data.country;
    this.postalCode = data.postalCode;
    this.personId = data.personId;
    this.customerId = data.customerId;
    this.isDeleted = data.isDeleted;
  }
}
